src/capitalsFirst.js

```diff
@@ -18,7 +18,13 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const filteredArr = str.split(' ').filter(el => (/[a-zA-Z]/g).test(el[0]));
```

Comment:
why do you do filtering?

Reply:
cause I wanna filter out strings that start with special symbols or numbers